libcxx/docs/Status/Cxx20Papers.csv

```diff
@@ -103,7 +103,7 @@
 "`P0466R5 <https://wg21.link/P0466R5>`__","Layout-compatibility and Pointer-interconvertibility Traits","Cologne","","",""
 "`P0553R4 <https://wg21.link/P0553R4>`__","Bit operations","Cologne","|Complete|","9.0",""
 "`P0631R8 <https://wg21.link/P0631R8>`__","Math Constants","Cologne","|Complete|","11.0",""
-"`P0645R10 <https://wg21.link/P0645R10>`__","Text Formatting","Cologne","|Complete| [#note-P0645]_","14.0",""
+"`P0645R10 <https://wg21.link/P0645R10>`__","Text Formatting","Cologne","|Complete| [#note-P0645]_","19.0",""
```

Comment:
I would remove this change. It is correct, but feels to me it a bit misleading. The paper was done, only the feature-test macro was combined with other papers.